### DISCUSSION TITLE
Cropper, where are the files saved?

### DISCUSSION THREAD

**Discussion by George-Fx** [orchidsoftware/platform#2286] [Q&A]
I'm trying to get a link that gives me an image cropper, but it doesn't work and there is no such image in the storage folder.

code: 
![Screenshot_8](https://user-images.githubusercontent.com/64652126/175275734-e5317e06-6c04-4c2d-9858-1477c3d18903.png)

**Comment by tabuna** [comment]
Images and other files are stored by default in laravel's default storage. Make sure you have a symbolic link and not a real directory. And also that the system user who replenished the script has the rights to create files in `storage/app/public`
